common/djangoapps/student/views.py

```diff
@@ -300,12 +300,13 @@ def _cert_info(user, course_overview, cert_status, course_mode):  # pylint: disa
 
     default_status = 'processing'
 
-    default_info = {'status': default_status,
-                    'show_disabled_download_button': False,
-                    'show_download_url': False,
-                    'show_survey_button': False,
-                    'can_unenroll': True
-                    }
+    default_info = {
+        'status': default_status,
+        'show_disabled_download_button': False,
+        'show_download_url': False,
+        'show_survey_button': False,
+        'can_unenroll': True
```

Comment:
Anywhere we can, we should always add trailing commas (especially when we're already editing the line).
For those of us living downstream, this makes it easier for us to extend code in our forks, while minimizing potential future conflicts.